README.md

```diff
@@ -66,6 +66,24 @@ The following steps outline the process of adding new variables to the project.
 
         new_variable: {{ new_variable }}
 
+### Removing old variables
+
+Removed variables will be documented in the [changelog](CHANGELOG.md) for each release. Review the [changelog](CHANGELOG.md) to ensure that all on wanted variables are removed from the configuration files.
+
+#### Encrypted
+
+1. Remove the variable form the `CREDENTIAL_CONFIG.yml` file.
+
+2. Insure the variable has been removed from the `bootstrap_credentials.yml`. This should have been done as part of the update.
```

Comment:
"This should have been done as part of the update." - not quite sure what is being referred to here? If a variable already exists it must be in the `bootstrap_credentials.yml` file?

Reply:
I have reworded this a small bit, hopefully its more understandable.